sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-calculation/e-cert-calculation-process.ts

```diff
@@ -18,6 +24,11 @@ interface GroupedDisbursementPerStudent {
   [studentId: number]: EligibleECertDisbursement[];
 }
 
+interface NotificationData {
+  max_created_at: Date;
```

Comment:
The other suggested refactor should make the name aligned with the convention `maxCreatedAt`.